### ISSUE TITLE
New bundled TypeScript typings do not expose various top-level exports

### ISSUE THREAD

**Issue by thw0rted** [pinojs/pino#1193]
As I commented in #910, the bundled TypeScript type definitions do not allow `import {destination, stdSerializers} from "pino"`, which is perfectly legal code.

Looking at `pino.js` (as shipped in v7.0.5), I see top level exports defined for the following variables:

````text
destination
transport
multistream
final
levels
stdSerializers
stdTimeFunctions
symbols
version
````

Since these can be imported from the module, the typings should include definitions for them, to accurately describe the shape of the JS module.

While you're at it, consider removing the namespace "P" that wraps all your type and interface declarations, and exporting them directly.  Namespace bundling is no longer considered a best practice in TS, and prevents the use of the (relatively recently introduced) `import type` syntax.

This has real-world consequences.  If I want to write a library that takes a `DestinationStream`, writing `import pino from "pino"; export function withStream(str: pino.DestinationStream) { ... }` results in including the entire `pino` library at runtime, even though I'm not actually constructing a logger or using any of pino's runtime logic, while the exact same code with `import type {DestinationStream} from "pino"` instead, only requires the library at compile-time, and doesn't bundle the pino implementation.

**Comment by KoltesDigital** [comment]
`import type {Logger} from "pino";` worked in 6.x but not in 7.x anymore. Bringing it back should be a _patch_ ... and of course I'd like to have it released as soon as possible so that I don't have to temporarily switch to P 🤗 . I guess many users of this library are in this case. I also think you can leave P with a deprecated mention, I guess few users switched.

FTR I found this thread because I'm upgrading a project's dependencies and tsc complains about non-existing Logger type, whereas it worked before.

**Comment by Retro64** [comment]
I stumbled over the same lost exported top-level Logger type when upgrading to the new version. I agree it should be a patch, as adding another top-level export does not mean the export within the namespace P needs to be dropped. It - for sure - becomes redundant, but won't be breaking (as it is for now for people, blindly upgrading the dependency to 7.x).

**Comment by kibertoad** [comment]
Released in 7.5.0
